src/main/java/Deadline.java

```diff
@@ -0,0 +1,14 @@
+public class Deadline extends Task {
+
+    protected String by;
+
+    public Deadline(String description, String by) {
+        super(description);
+        this.by = by;
+    }
+
+    @Override
+    public String toString() {
```

Comment:
I like the use of toString() to print details of the task. The method name itself is already specific to its action and it is a built-in method in all java objects.